site/sfguides/src/tealium-snowflake-integration/tealium-snowflake-integration.md

```diff
@@ -0,0 +1,463 @@
+- **Timestamp**: Tealium imports new or modified rows based on a timestamp column. In this mode, rows with a newer timestamp than the previous import are imported. Use this mode if your table has a timestamp column that gets set or updated on every update.
+- **Incrementing**: Tealium imports rows based on an auto-increment column. In this mode, rows with a larger auto-increment value than the last imported row are imported. This will not detect modifications or deletions of existing rows. Use this mode if you only have an auto-increment column and do not have a timestamp column.
+
+The Snowflake data source does not support bulk query modes.
+
+#### Query Mode Example
+The following example shows how batch processing of rows and query modes work together. In the following table, **modification_time** is the timestamp column and **customer_id** is the incrementing column.
+
+| **customer_id** | **modification_time** | **customer_segment** |
+| --------------- | --------------------- | -------------------- |
+| 1 | 01Apr 13:00 | A |
+| 2 | 01Apr 13:00 | B |
+| ... | ... | ... |
+| 1000 | 01Apr 13:00 | D |
+| 1001 | 01Apr 13:00 | E |
+| 1002 | 02Apr 14:00 | A |
+
+The Snowflake data source fetches data 1,000 rows at a time and marks the maximum value of the timestamp and/or incrementing column(s) from the batch of data.
+
+- Using **Timestamp + Incrementing** mode: The data source fetches rows 1-1000. The next time the data source fetches the data, it will look for rows where either
+````
+modification_time is 01Apr 13:00 and the customer_id is greater than 1000
+````
+or
+````
+modification_time is greater than 01Apr 13:00
+````
+- Using **Timestamp** mode (**modification_time** in the example): The data source will fetch rows 1-1000 and mark the maximum timestamp of **01Apr 13:00**. The next time the data source fetches the data, it will look for a timestamp greater than **01Apr 13:00**. In this case, row 1001 would be skipped because it has the same timestamp value but was fetched in different batch of data.
+Using **Incrementing** mode: The data source will fetch rows 1-1000 and mark the maximum auto-increment value of **1000**, but rows with new data and an updated timestamp column would not be processed. Only newly added rows that further increment the incrementing column (customer_id in the example) would be processed.
+
+#### SQL Query
+In the Snowflake data source **Query Configuration**, select the columns you want to import into Tealium. To add additional conditions for processing, use the SQL **WHERE** clause. This option adds a **WHERE** statement to your query. **WHERE** statements support basic SQL syntax.
```

Comment:
Screenshots would be helpful here.

Reply:
We are currently updating UI around customer feedback and will include screenshots in a future update to this guide.